/**:
  ros__parameters:
    common:
      lid_topic: "/ouster/points"
      imu_topic: "/ouster/imu"
      time_sync_en: false
      time_offset_lidar_to_imu: 0.0

    preprocess:
      lidar_type: 3 # 1 for Livox, 2 for Velodyne, 3 for ouster
      scan_line: 32
      timestamp_unit: 3 # 0-second, 1-milisecond, 2-microsecond, 3-nanosecond
      blind: 2.0

    mapping:
      acc_cov: 0.1
      gyr_cov: 0.1
      b_acc_cov: 0.0001
      b_gyr_cov: 0.0001
      fov_degree: 180.0
      det_range: 150.0
      extrinsic_est_en: false
      extrinsic_T: [0.0, 0.0, 0.0]
      extrinsic_R: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]

    publish:
      path_en: false
      scan_publish_en: true
      dense_publish_en: true
      scan_bodyframe_pub_en: true

    ivox_grid_resolution: 0.5
    ivox_nearby_type: 18
